spec/models/task_spec.rb

```diff
@@ -23,8 +23,9 @@
   end
 
   describe '検索機能' do
-    let!(:task) { FactoryBot.create(:testtask) }
-    let!(:second_task) { FactoryBot.create(:testtask_second, title: 'sample') }
+    let!(:factory_user) { FactoryBot.create(:user) }
+    let!(:task) { FactoryBot.create(:testtask, user: factory_user) }
+    let!(:second_task) { FactoryBot.create(:testtask_second, title: 'sample', user: factory_user) }
```

Comment:
Factoryの定義については設定ファイルに追記することで省略することが可能です。
```suggestion
    let!(:factory_user) { create(:user) }
    let!(:task) { create(:testtask, user: factory_user) }
    let!(:second_task) { create(:testtask_second, title: 'sample', user: factory_user) }
```

Reply:
修正済みです。 2db29e1